### PR TITLE
Create dark mode for Featherlight

### PR DESCRIPTION
**Pull request by int-y1**
Part of #2035. Move `featherlight.min.css` and inline css into a single scss file.

Old:
Capture

New:
Capture

### CODE REVIEW

File: resources/featherlight.scss [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    }
+
+    &:before {
+        content: '';
+        display: inline-block;
+        height: 100%;
+        vertical-align: middle;
+    }
+
+    & .featherlight-content {
```
Comment (quantum5) [inline resources/featherlight.scss:26]: Why do you need `& .`?

Comment (int-y1) [reply resources/featherlight.scss:26]: Featherlight used `.featherlight .featherlight-content`, and I wrote it here as `& .featherlight-content`.

Comment (quantum5) [reply resources/featherlight.scss:26]: This is redundant. `.featherlight-content` works just fine. It nests by default and `&` is only needed when it's on the same level.

Comment (int-y1) [reply resources/featherlight.scss]: Ok, cleaned up.

File: resources/featherlight.scss [hunk 2]
```diff
@@ -0,0 +1,91 @@
+@import "vars";
+
+// Based on Featherlight 1.7.14
+
+.featherlight {
+    display: none;
+    position: fixed;
+    top: 0; right: 0; bottom: 0; left: 0;
+    z-index: 2147483647;
```
Comment (quantum5) [inline resources/featherlight.scss:9]:
```suggestion
    z-index: 1000;
```

Let's not go straight to the maximum. What if we wanted to display something else on top?

**Review by quantum5 — APPROVED** [review]
Doesn't seem broken.

**Pull request merged**